src/Uno.UI.Composition/Composition/CompositionObject.Context.cs

```diff
@@ -7,118 +7,110 @@ namespace Microsoft.UI.Composition
 {
 	public partial class CompositionObject
 	{
-		private class ContextStore
```

Comment:
A nested class was separating the store from the composition object quite well. I personally prefer that separation over making `CompositionObject` class more large.

If there are performance concerns, we might make it a struct.

@dr1rrb What do you think?

Reply:
IMHO, it doesn't make sense to have a nested class that has the same lifetime as it's outer class and most of its methods have corresponding "passthrough methods" in the outer class. It just creates unnecessary indirections.